woocommerce/woocommerce
# Add e2e test to check ability to add and remove regions from shipping…

… zone

### All Submissions:






### Changes proposed in this Pull Request:



Closes [#32308](https://github.com/woocommerce/woocommerce/issues/32308).

Adds a test which will add and remove a Region within a zone and validate accordingly

### How to test the changes in this Pull Request:

1. Navigate to https://github.com/woocommerce/woocommerce/tree/trunk/plugins/woocommerce/e2e
2. Follow the https://github.com/woocommerce/woocommerce/tree/trunk/plugins/woocommerce/e2e#:~:text=ways%20of%20running%20tests to execute
3. Ensure the tests pass

### Other information:




### FOR PR REVIEWER ONLY:
